libbeat/generator/fields/module_fields_collector.go

```diff
@@ -0,0 +1,61 @@
+package fields
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+var indentByModule = map[string]int{
+	"processors": 0,
+	"module":     8,
```

Comment:
It's a bit a pity that libbeat has knowledge here outside the scope of libbeat. But probably best way to go for now.